src/FileContents/InspectionsXml.php

```diff
@@ -138,7 +138,7 @@ public function getName(): string
         return $this->name;
```

Comment:
remove the getName methods if not needed. Also think about whether to have the (public) const for name or not